net/uacme/files/run.sh

```diff
@@ -0,0 +1,396 @@
+#!/bin/sh
```

Comment:
I see several shellcheck warnings. Would be great to look at them.

Reply:
I've tested all of them before last commit, the script will not work if I follow the recommendations

Reply:
You have to be real careful `shellcheck`.  I've tried it here and it helped me to catch a bug, but it was the actual opposite of what it suggested.  It did, at the same time, point out useful changes.

Reply:
You have to be careful with `shellcheck`.  I've tried it here and it kind of helped me (indirectly) to catch a bug, but I did the opposite of what `shellcheck` thinks is right.  It did, at the same time, point out useful changes.  We just can't follow it blindly.

Reply:
In my experience, it's overzealous when it comes to quoting warnings. Otherwise, it tends to work fine.